components/style/properties/shorthand/position.mako.rs

```diff
@@ -495,16 +495,30 @@
         })
     }
 
+    /// Returns true if every sub property value of `grid` shorthand is initial.
+    impl<'a> LonghandsToSerialize<'a> {
+        fn is_initial(&self) -> bool {
```

Comment:
I'm not happy about this method but this seems the best way to check it. Without that we can't serialize this shorthand correctly. 😕